lib/arsi.rb

```diff
@@ -53,10 +55,11 @@ def enable(&block)
     private
 
     def arel_where_sql(arel, relation)
-      return arel.where_sql if relation.nil?
-      return arel.where_sql unless relation.model.respond_to?(:arel_engine)
-
-      arel.where_sql(relation.model.arel_engine)
+      if relation.nil?
```

Comment:
I did not want to change it in this PR to avoid expanding the scope further, and it would be a breaking change, but it seems like we could simply replace `relation` with `model`.

Reply:
you dropped arel_engine tho?

Reply:
arel_engine is removed between 5.1 and 5.2, so we no longer need to support that case.